esti/delete_objects_test.go

```diff
@@ -42,7 +42,52 @@ func TestDeleteObjects(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Len(t, deleteOut.Deleted, numOfObjects)
 
-	listOut, err = svc.ListObjects(&s3.ListObjectsInput{
+	listOut, err = pathStyleSvc.ListObjects(&s3.ListObjectsInput{
+		Bucket: aws.String(repo),
+		Prefix: aws.String(mainBranch + "/"),
+	})
+
+	assert.NoError(t, err)
+	assert.Len(t, listOut.Contents, 0)
+}
+
+func TestDeleteObjectsHostStyleSvc(t *testing.T) {
```

Comment:
Instead of copying the entire test which may lead to future bugs, you can write it as a function that accepts the S3 client and uses that throughout the test. Then simply trigger that function with the appropriate client when needed